src/ProjectService.Data/Interfaces/IProjectRepository.cs

```diff
@@ -45,7 +46,7 @@ public interface IProjectRepository
         /// </summary>
         /// <param name="showNotActive">Do you want to show inactive projects?</param>
         /// <returns>All projects.</returns>
-        IEnumerable<DbProject> GetProjects(bool showNotActive);
+        //IEnumerable<DbProject> GetProjects(bool showNotActive);
```

Comment:
why comment?

Reply:
now it deleted, before i didn't want to do it